src/components/Books.js

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react';
+import defaultBooks from '../defaultBooks';
+
+const Books = () => {
+  const [bookList] = useState(JSON.parse(localStorage.getItem('books')) || defaultBooks);
+
+  return (
+    <>
+      <header>
+        <h1>Books</h1>
+      </header>
+      <main>
+        <section>
+          <ul>
+            {bookList.map((book) => (
+              <li key={book.title}>
+                <article>
+                  <span>{book.category}</span>
+                  <h1>{book.title}</h1>
+                  <data>{book.author}</data>
+                  <button type="button">Remove</button>
+                </article>
```

Comment:
As specified in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/be3b13a85ca425c0bf0eeb1f130b6a52e36deb4f/bookstore/project_init_project_wth_comps_v3_0.md?plain=1#L42), kindly create a new component for book and use it within the books component.